src/main/java/org/folio/dcb/service/impl/EcsRequestTransactionsServiceImpl.java

```diff
@@ -70,6 +71,29 @@ public TransactionStatusResponse createEcsRequestTransactions(String ecsRequestT
     }
   }
 
+  @Override
+  public TransactionStatusResponse updateEcsRequestTransactions(String ecsRequestTransactionsId, DcbTransaction dcbTransaction) {
+    log.info("updateEcsRequestTransactions:: updating transaction {} for role {} ",
```

Comment:
Does it make sense to log role here? It won't have role when only barcode is updated